tests/phpunit/Civi/Token/TokenProcessorTest.php

```diff
@@ -45,19 +44,23 @@ public function testVisitTokens() {
       '{foo.bar}' => ['foo', 'bar', NULL],
       '{foo.bar|whiz}' => ['foo', 'bar', ['whiz']],
       '{foo.bar|whiz:"bang"}' => ['foo', 'bar', ['whiz', 'bang']],
-      '{love.shack|place:"bang":"b@ng, on +he/([do0r])?!"}' => ['love', 'shack', ['place', 'bang', 'b@ng, on +he/([do0r])?!']],
```

Comment:
you didn't like this one?

Reply:
Well, I wouldn't want anyone to accuse of desecrating the art of the B-52s...

Reply:
lol